runtime/queries/gpr/highlights.scm

```diff
@@ -0,0 +1,51 @@
+[ "abstract" "all" "at"
+  "case"
+  "end" "extends" "external" "external_as_list"
+  "for"
+  "is"
+  "limited"
+  "null"
+  "others"
+  "package"
+  ;; "project"
+  "renames"
+  "type"
+  "use"
+  "when"
+  "with"
+  ] @keyword
+
+;; Avoid highlighting Project in Project'Project_Dir
+(project_declaration "project" @keyword)
+
+;; highlight qualifiers as keywords (not all qualifiers are actual keywords)
+(project_qualifier _ @keyword)
+
+[":=" "&" "|" "=>"] @operator
+
+(comment) @comment
+(string_literal) @string
+(numeric_literal) @number
```

Comment:
```suggestion
(numeric_literal) @constant.numeric
```

We use slightly different capture names than neovim's (which I believe is what these were originally written for) https://docs.helix-editor.com/master/themes.html#syntax-highlighting. This is the only one I see that needs fixing though.